parachain/pallets/eth-app/src/lib.rs

```diff
@@ -90,18 +87,14 @@ decl_module! {
 
 			T::Asset::withdraw(&who, amount)?;
 
-			let message = OutPayload {
+			let message = OutboundPayload {
 				sender_addr: who.clone(),
 				recipient_addr: recipient,
 				amount: amount
 			};
-			T::Commitments::add(Self::address(), message.encode());
-
+			T::MessageCommitment::add(Self::address(), 0, &message.encode());
```

Comment:
As above, should this not be incrementing?

Reply:
That was just a placeholder for the nonce handling I've implemented in this follow-up PR: https://github.com/Snowfork/polkadot-ethereum/pull/223/files#diff-a3f0c878bcb36ffd0b0fabe45b220da7797857e800a53208b5fce46161dd50a4R41

Reply:
👍